index.html

```diff
@@ -114,8 +114,15 @@
   case ${url} in
     *tar.xz)
       ${get} "${url}" | tar xfJ - || return 1
-      # shellcheck disable=SC2086
-      install_subctl subctl*/subctl*${os}-${architecture}*
+      version=$(echo "$url" | grep -oE 'v[0-9]+\.[0-9]+\.[0-9]+|devel' | sed 's/^v//')
+      # Compare the version with 0.16.0 or "devel"
+      if [[ "$version" == "devel" || "$(printf '%s\n' "0.16.0" "$version" | sort -V | head -n1)" == "0.16.0" ]]; then
+        # shellcheck disable=SC2086
+        install_subctl subctl*/subctl
+      else
+        # shellcheck disable=SC2086
+        install_subctl subctl*/subctl*${os}-${architecture}*
+      fi
```

Comment:
Given that each tarball ships a single binary, I think we can get away with

```suggestion
      install_subctl subctl*/subctl*
```

This avoids having to deal with 0.16.0 pre-releases too ;-).